_sass/tiendatek-theme.scss

```diff
@@ -683,6 +692,113 @@ footer {
   font-family: 'MontserratRegular';
 }
 
+// Mobile view
+@include media-breakpoint-down(sm) {
+  .testimonial {
+    font-size: 1em;
+    padding: 1.5em;
+    h1 {
+      margin-bottom: 1em;
+    }
+    iframe {
+      width: 100%;
+    }
+    .img-fluid {
```

Comment:
Same here, a horrible SoC between Bootstrap and our styles... Janitors material...